From 0f5a0c16040f85a6b52c6c8f741ffd41cbd8f632 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Mon, 16 Nov 2020 08:38:15 +0200
Subject: [PATCH] [SW-5080] version: replace CFLAGS for target

Since we use here native compiler do not use the flags for CC

Change-Id: I5d1bcdc11de1bb29f42e1e9a592269fff8d7544e
---

diff --git a/Makefile b/Makefile
index 910af4a..620d405 100644
--- a/Makefile
+++ b/Makefile
@@ -76,7 +76,7 @@
 	@date +'#define PREBOOT_TIME "%T"' >> $@
 
 buildversion-habanalabs: include/preboot.h $(AUTO_GENERATED_FILE)
-	echo "VERSION_STR" | gcc -include $< $(CFLAGS) -P -E -o - - | tail -1 | sed 's/\" \+\"//g;s/["\n]//g' | cut -d' ' -f2- | tr -d '\n' > $@
+	echo "VERSION_STR" | gcc -include $< -I./ -P -E -o - - | tail -1 | sed 's/\" \+\"//g;s/["\n]//g' | cut -d' ' -f2- | tr -d '\n' > $@
 
 $(PCI_PHY_FW_TARGET): $(HL_DEV)/pcie_phy_fw.i
 	@cut -d' ' -f2- $< | xxd -r -p | dd status=none bs=16 count=$$(($(FW_SIZE) / 16)) conv=sync,swab of=$@
